src/main/java/com/hazelcast/commandline/HazelcastCommandLine.java

```diff
@@ -0,0 +1,99 @@
+package com.hazelcast.commandline;
+
+import com.hazelcast.commandline.member.MemberCommandLine;
+import picocli.CommandLine;
+
+import java.io.PrintStream;
+import java.util.List;
+import java.util.concurrent.Callable;
+
+import static com.hazelcast.instance.BuildInfoProvider.getBuildInfo;
+import static picocli.CommandLine.*;
+
+@Command(
+        name = "hazelcast",
+        description = "Utility for the Hazelcast IMDG operations." +
+                "%n%n" +
```

Comment:
Sorry for ignorance, I'm new to Picocli, where are these "%n" filled?

Reply:
No worries, they are special newline chars.